v2/paths/recognition/GroupsSearch.js

```diff
@@ -0,0 +1,75 @@
+module.exports = {
+  get: {
```

Comment:
As mentioned before, this endpoint is `POST` and params are expected to be in the request body rather than as a query string.

Reply:
This is resolved